dfinity/nns-dapp
# NNS1-2851: Remove loadSnsAccountTransactions

# Motivation

Unify wallet implementation.

In this PR, remove `loadSnsAccountTransactions` service.

# Changes

* Remove flag `loadTransactions` in SnsTransactionModal that was used only in wallet page.
* Remove flag `loadTransactions` in `snsTransferTokens`.
* Remove file sns-transactions services.

# Tests

* Remove test `"should load transactions if flag is passed"` for `snsTransferTokens`.
* Remove spec file sns-transactions services.

# Todos


Not necessary.
